students/julienjass/Lesson04/Mailroom2.py

```diff
@@ -0,0 +1,146 @@
+#!/usr/bin/env python3
+
+import sys
+
+# Data base made of a dictionary containing the donors' information
+donors = {
+    'William Gates, III': [261514, 392270.49],
+    'Mark Zuckerberg': [4918.83, 9837.66, 1639.61],
+    'Jeff Bezos': [877.33],
+    'Paul Allen': [354.21, 212.53, 141.68]
+    }
+
+
+def mainloop():
+    """Main menu"""
+    print('Mailroom Application')
+    response = ''
+    switch_dict = {
+        '1': thank_you,
+        '2': report,
+        '3': write_letters_on_disk,
+        '4': leave
+    }
+    while response != '4':
+        print('Please select one of the 3 options:')
+        print('''
+        1) Send a Thank You
+        2) Create a Report
+        3) Creating letters for all donors
+        4) Quit''')
+        print()
+        response = input('Your answer: ')
+        try:
+            switch_dict.get(response)()
+        except TypeError:
+            print('This is not a valid response. Please type either 1, 2, 3, or 4\n')
+        continue
```

Comment:
you don't need the `continue` here since the loop will natural continue looping and you don't have any additional code after it